phy/cluster/algorithms.py

```diff
@@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
     def _create_filter(self):
         rate = self._kwargs['sample_rate']
         low = self._kwargs['filter_low']
```

Comment:
I realize that I forgot to explain the motivation for the change: having a `sample_rate` variable in the parameter files makes the code much more complicated.  You can't have a simple dictionary for the parameters, you need a function of the `sample_rate` variable. That's why I switched to a factor of the sample rate.